lib/statsd/instrument/metric_expectation.rb

```diff
@@ -0,0 +1,30 @@
+class StatsD::Instrument::MetricExpectation < StatsD::Instrument::Metric
+
+  attr_accessor :times, :ignore_tags
+
+  # Initializes a new MetricExpectation instance.
+  #
+  # @times times [Numeric] :times The number of times this metric is expected
+  # @@ignore_tags @ignore_tags [Array<String>, Hash<String, String>, nil] :@ignore_tags The tags
+  #   to ignore for this metric.
+  def initialize(options = {})
+    super(options)
+    @times = options[:times] or raise ArgumentError, "Metric :times is required."
```

Comment:
hm this is conventionally done as `@times = options.fetch(:times)` (which raises if the key doesn't exist)